includes/widgets/table.php

```diff
@@ -104,14 +103,14 @@ protected function get_upsale_data() {
 			'condition' => ! Utils::has_pro(),
 			'image' => esc_url( ELEMENTOR_ASSETS_URL . 'images/go-pro.svg' ),
 			'image_alt' => esc_attr__( 'Upgrade', 'elementor' ),
-			'description' => esc_html__( 'Create captivating testOnes that rotate with the Animated Headline Widget.', 'elementor' ),
-			'upgrade_url' => esc_url( 'https://go.elementor.com/go-pro-testOne-widget/' ),
+			'description' => esc_html__( 'Create captivating tables that rotate with the Animated Headline Widget.', 'elementor' ),
```

Comment:
`'description' => esc_html__( 'Create captivating tables.', 'elementor' ),`

Reply:
I changed the description